### ISSUE TITLE
Nextcloud related configuration

### ISSUE THREAD

**Issue by dg3feh** [Rillke/Contact-Conversion-Wizard#16]
The check box on the configuration page is not shown correctly. After closing the window, the status disapers.

**Comment by tryallthethings** [comment]
You are correct. The Nextcloud setting isn't saved properly. Will be fixed in the next version.
